index.js

```diff
@@ -31,7 +31,7 @@ function createFactory(moduleName) {
         options = mockOptions;
       }
       const LoadableComponent = dynamic(loader, options);
-      mockInitializers.push(() => LoadableComponent.preload());
+      mockInitializers.push(() => LoadableComponent.render.preload());
```

Comment:
We don't want to be tied to an exact Next.js version necessarily – how about checking for both `LoadableComponent.preload` and `LoadableComponent.preload .render.preload`?